src/QMCWaveFunctions/tests/test_soa_cusp_corr.cpp

```diff
@@ -528,9 +528,9 @@ TEST_CASE("Ethanol MO with cusp", "[wavefunction]")
   SPOSet::ValueMatrix all_values;
   SPOSet::GradMatrix all_grad;
   SPOSet::ValueMatrix all_lap;
-  all_values.resize(13, 13);
-  all_grad.resize(13, 13);
-  all_lap.resize(13, 13);
+  all_values.resize(10, 13);
```

Comment:
Where does the "10" come from?
The following call to "evaluate_notranspose" uses 0,7 for first,last.  For ethanol, that should be 0,14. I think the code was copied from the HCN test and those values were not updated to match.  However, it could be left as-is to test a case where there are fewer electrons than orbitals.

Reply:
You can consider it as a non-ground state calculation with only 10 electron in one spin.

Reply:
From evaluate_notranspose API with first, last, there is no restriction requiring the container size to match exactly. So 0.7 can be considered as a corner case test.

Reply:
@markdewing looks good?

Reply:
Yes, keeping these values to test non-standard cases is good.